ultralytics/utils/export.py

```diff
@@ -27,9 +27,9 @@ def export_onnx(
         im (torch.Tensor): Example input tensor for the model.
         onnx_file (str): Path to save the exported ONNX file.
         opset (int): ONNX opset version to use for export.
-        input_names (List[str]): List of input tensor names.
-        output_names (List[str]): List of output tensor names.
-        dynamic (bool | Dict, optional): Whether to enable dynamic axes.
+        input_names (list[str]): List of input tensor names.
+        output_names (list[str]): List of output tensor names.
+        dynamic (bool | dict, optional): Whether to enable dynamic axes.
```

Comment:
@onuralpszr we need to check if Google-style docstrings use ", optional" or "| None" and then be consistent.